tutorials/performance/using_multiple_threads.rst

```diff
@@ -47,6 +47,94 @@ To create a thread, use the following code:
     func _exit_tree():
         thread.wait_to_finish()
 
+ .. code-tab:: cpp C++ .H File
+
+    #ifndef MULTITHREADING_DEMO_H
+    #define MULTITHREADING_DEMO_H
+
+    #include <godot_cpp/classes/node.hpp>
+    #include <godot_cpp/classes/thread.hpp>
+
+    namespace godot {
+        class MultithreadingDemo : public Node {
+            GDCLASS(MultithreadingDemo, Node);
+
+        private:
+            Ref<Thread> worker;
+
+        protected:
+            static void _bind_methods();
+            void _notification(int p_what);
+
+        public:
+            MultithreadingDemo();
+            ~MultithreadingDemo();
+
+            void demo_threaded_function();
+        };
+    } // namespace godot
+
+    #endif // MULTITHREADING_DEMO_H
+
+ .. code-tab:: cpp C++ .CPP File
+
+    #include "multithreading_demo.h"
+
+    #include <godot_cpp/classes/engine.hpp>
+    #include <godot_cpp/classes/os.hpp>
+    #include <godot_cpp/classes/time.hpp>
+    #include <godot_cpp/core/class_db.hpp>
+    #include <godot_cpp/variant/utility_functions.hpp>
+
+    using namespace godot;
+
+    void MultithreadingDemo::_bind_methods() {
+        // THREADING
+        ClassDB::bind_method(D_METHOD("threaded_function"), &MultithreadingDemo::demo_threaded_function);
+    }
+
+    void MultithreadingDemo::_notification(int p_what) {
+        // Prevents this from running in the editor, only during game mode.
+        if (Engine::get_singleton()->is_editor_hint()) {
+            return;
+        }
```

Comment:
If the nodes are only used during runtime it would make sense to add a note to use Runtime classes in Godot 4.3+